### PR TITLE
new cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VI Instrument Cal CSVs [DT-219]](https://trello.com/c/CkLoKMMl/219-py-vi-instrument-cal-csvs-dt-219)

File: calibration/OPTAAD/CGINS-OPTAAD-00207__20231218__CC_taarray.ext [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+-0.000503,-0.000465,-0.000484,-0.000470,-0.000812,-0.000999,-0.000674,-0.000131,0.000024,-0.000223,-0.000107,-0.000306,-0.000582,-0.000601,0.000004,-0.000321,-0.000527,-0.000594,-0.000268,-0.000458,-0.000344,-0.000370,-0.000330,0.000182,0.000180,0.000000,-0.000108,-0.000090,-0.000090,-0.000123,-0.000024,-0.000201,0.000074,0.000083,0.000044,-0.000096
+-0.000362,-0.000371,-0.000524,-0.000477,-0.000861,-0.001041,-0.000735,-0.000054,-0.000044,-0.000178,-0.000145,-0.000474,-0.000723,-0.000836,-0.000173,-0.000502,-0.000536,-0.000658,-0.000262,-0.000470,-0.000371,-0.000521,-0.000417,0.000138,0.000075,0.000000,-0.000192,-0.000115,-0.000342,-0.000308,-0.000099,-0.000261,0.000064,0.000048,-0.000022,-0.000476
+-0.000333,-0.000329,-0.000453,-0.000520,-0.000796,-0.001095,-0.000666,-0.000028,0.000052,-0.000151,-0.000067,-0.000376,-0.000658,-0.000814,-0.000053,-0.000389,-0.000560,-0.000553,-0.000222,-0.000469,-0.000391,-0.000577,-0.000360,0.000203,0.000123,0.000000,-0.000076,-0.000045,-0.000181,-0.000238,-0.000012,-0.000305,0.000059,0.000122,-0.000012,-0.000109
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00207__20231218__CC_taarray.ext:86]: spot checked - looks good

**Review by cdobs — CHANGES_REQUESTED** [review]
Just one small change requested: update source file name in CGINS-CTDBPD-50110__20231107.csv

File: calibration/CTDBPD/CGINS-CTDBPD-50110__20231107.csv [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+16-50110,CC_pa2,8.118608e-011,
+16-50110,CC_ptca0,5.210080e+005,
+16-50110,CC_ptca1,-6.876503e+001,
+16-50110,CC_ptca2,8.168275e-002,
+16-50110,CC_ptcb0,2.480888e+001,
+16-50110,CC_ptcb1,-2.250000e-004,
+16-50110,CC_ptcb2,0.000000e+000,
+16-50110,CC_ptempa0,1.789140e+002,
+16-50110,CC_ptempa1,-7.860039e+001,
+16-50110,CC_ptempa2,9.283013e-001,
```
Comment (cdobs) [inline calibration/CTDBPD/CGINS-CTDBPD-50110__20231107.csv:23]: updated source file looks good!

File: calibration/DOSTAD/CGINS-DOSTAD-00428__20231129.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+428,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_428_Multipoint_Calibration_2023-11-29.pdf"
+428,CC_csv,"[2.46260E-03, 1.04744E-04, 2.04518E-06, 1.66805E+02, -2.19485E-01, -4.05650E+01, 3.37945E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00428__20231129.csv:3]: updated format looks good!

**Review by cdobs — APPROVED** [review]

File: calibration/OPTAAD/CGINS-OPTAAD-00207__20231218__CC_taarray.ext (hunk 1 above)
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00207__20231218__CC_taarray.ext:86]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00207__20231218__CC_tcarray.ext [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+0.009604,0.011643,0.013722,0.012719,0.012390,0.012183,0.010458,0.009959,0.009375,0.008680,0.008151,0.008076,0.007442,0.006836,0.006284,0.005839,0.005319,0.005051,0.004210,0.003845,0.003042,0.002639,0.001804,0.001148,0.000488,0.000000,-0.000590,-0.001140,-0.001770,-0.002191,-0.002720,-0.003517,-0.004176,-0.004776,-0.005510,-0.005328
+0.011616,0.013302,0.014233,0.013278,0.012847,0.012737,0.011068,0.010272,0.009622,0.009151,0.008754,0.008125,0.007694,0.007112,0.006391,0.005951,0.005534,0.005125,0.004485,0.004003,0.003270,0.002554,0.001978,0.001291,0.000760,0.000000,-0.000293,-0.001198,-0.001678,-0.002560,-0.003115,-0.003657,-0.004454,-0.004937,-0.005589,-0.006220
+0.008608,0.011053,0.014258,0.013475,0.013021,0.012689,0.010997,0.010267,0.009886,0.009345,0.008723,0.008077,0.007507,0.007101,0.006643,0.006313,0.005463,0.005085,0.004403,0.004027,0.003237,0.002550,0.001820,0.001272,0.000493,0.000000,-0.000528,-0.001234,-0.001871,-0.002560,-0.003088,-0.003667,-0.004562,-0.005034,-0.005827,-0.006630
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00207__20231218__CC_tcarray.ext:86]: spot checked, looks good

**Pull request merged**